torch/onnx/_internal/fx/diagnostics.py

```diff
@@ -71,12 +78,52 @@ def _torch_fx_graph_module(obj: torch.fx.GraphModule) -> str:
 
 @_format_argument.register
 def _torch_fx_node(obj: torch.fx.Node) -> str:
-    return f"fx.Node({obj.name}[{obj.op}]'{obj.target}')"
+    node_string = f"fx.Node({obj.target})[{obj.op}]:"
+    if "val" not in obj.meta:
+        return node_string + "None"
+    return node_string + _format_nested_argument_by_dtype(obj.meta["val"])
+
+
+@_format_argument.register
+def _torch_fx_symbolic_value(
+    obj,  # NOTE: functools.singledispatch does not support Union until 3.11, so we use Any here.
```

Comment:
I wonder if a silly approach of creating 3 different functions for it is viable :)

Reply:
should then allow `format_argument` to be used instead of `_format_nested_argument_by_dtype`

Reply:
Done